packages/fluentui/react-northstar/gulpfile.ts

```diff
@@ -4,5 +4,5 @@ import { parallel, series, task } from 'gulp';
 import '../../../gulpfile';
 
 // Redefine test and test:watch to build info json files beforehand
-task('test', series('build:docs:component-info', 'test:jest'));
-task('test:watch', series('build:docs:component-info', parallel('test:jest:watch', 'watch:docs:component-info')));
+task('test', series('build:component-info', 'test:jest'));
```

Comment:
does this still need to happen here if it's happening in `build` ?

Reply:
Yes, as in Northstar there is no requirement to build before test run